db/migrate/20170531094819_move_conferences_to_organizations.rb

```diff
@@ -0,0 +1,21 @@
+class MoveConferencesToOrganizations < ActiveRecord::Migration
+  class TempConference < ActiveRecord::Base
+    self.table_name = 'conferences'
+  end
+
+  class TempOrganization < ActiveRecord::Base
+    self.table_name = 'organizations'
+  end
+
+  def change
+    add_reference :conferences, :organization, index: true
+    add_foreign_key :conferences, :organizations, dependent: :delete
+
+    TempConference.reset_column_information
+    organization = TempOrganization.create(name: 'organization', description: 'Default organization to migrate old conferences to the new version of OSEM')
+    TempConference.all.each do |conference|
+      conference.organization_id = organization.id
+      conference.save!
+    end
```

Comment:
Rubocop warns not to use `update_all` because it skips model based validations.

https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/rails/skips_model_validations.rb#L34